src/scene/materials/standard-material-options-builder.js

```diff
@@ -370,6 +385,12 @@ class StandardMaterialOptionsBuilder {
                 }
             }
         }
+
+        const litOptionsName = 'use' + mname[0].toUpperCase() + mname.substring(1);
```

Comment:
if these map options are changing anyway, would it make sense to group each map into its own object? (A bit like the standard material should have done).

So instead of having:
options.useDiffuseMap
options.diffuseMap
options.diffuseMapTransform
options.diffuseMapUv

make it:
options.diffuseMap.enabled
options.diffuseMap.texture
options.diffuseMap.transform
options.diffuseMap.uv
etc

Though now I'm wondering why these map options are part of litOptions in the first place. Aren't these all frontend?

Reply:
Done. What bothers me a little still (but we can deal with that in another pass over this) is that we only need a certain subset of texture usage info in the lit shader, but this code effectively adds that for all of them.

Reply:
Actually reverted this seeing as the lit options should not have all texture options but rather only needs a few flags from the frontend.